### ISSUE TITLE
Compile error Swift 4.1 Ubuntu

### ISSUE THREAD

**Issue by joscdk** [vapor/toolbox#218]
Toolbox don't compile on Ubuntu.

Tested on:

Swift: `4.1`
Ubuntu: `16.04`

```
Compile Swift Module 'Cloud' (73 sources)
/root/toolbox/Sources/Cloud/Commands/Beta/GroupBeta.swift:12:9: warning: initialization of immutable value 'addons' was never used; consider replacing with assignment to '_' or removing it
    let addons = Group(id: "addon", commands: [
    ~~~~^~~~~~
    _
/root/toolbox/Sources/Cloud/Commands/Beta/S3.swift:27:13: warning: initialization of immutable value 'app' was never used; consider replacing with assignment to '_' or removing it
        let app = try console.application(for: arguments, using: cloudFactory)
        ~~~~^~~
        _
/root/toolbox/Sources/Cloud/Commands/Configs/ConfigModify.swift:52:32: warning: 'characters' is deprecated: Please use String or Substring directly
            let parts = config.characters.split(separator: "=", maxSplits: 1)
                               ^
/root/toolbox/Sources/Cloud/Commands/Configs/ConfigModify.swift:60:22: warning: 'init' is deprecated: Please use String or Substring directly
                key: String(parts[0]),
                     ^
/root/toolbox/Sources/Cloud/Commands/Configs/ConfigModify.swift:61:24: warning: 'init' is deprecated: Please use String or Substring directly
                value: String(parts[1])
                       ^
/root/toolbox/Sources/Cloud/Commands/Configs/ConfigModify.swift:52:32: warning: 'characters' is deprecated: Please use String or Substring directly
            let parts = config.characters.split(separator: "=", maxSplits: 1)
                               ^
/root/toolbox/Sources/Cloud/Commands/Configs/ConfigModify.swift:60:22: warning: 'init' is deprecated: Please use String or Substring directly
                key: String(parts[0]),
                     ^
/root/toolbox/Sources/Cloud/Commands/Configs/ConfigModify.swift:61:24: warning: 'init' is deprecated: Please use String or Substring directly
                value: String(parts[1])
                       ^
/root/toolbox/Sources/Cloud/Commands/Database/DatabaseCreate.swift:45:9: warning: no calls to throwing functions occur within 'try' expression
        try environments.forEach { val in
        ^
/root/toolbox/Sources/Cloud/Commands/Database/DatabaseFeedbackFormat.swift:7:15: warning: 'catch' block is unreachable because no errors are thrown in 'do' block
            } catch {
              ^
/root/toolbox/Sources/Cloud/Commands/Database/DatabaseInspect.swift:46:9: warning: no calls to throwing functions occur within 'try' expression
        try environments.forEach { val in
        ^
/root/toolbox/Sources/Cloud/Commands/Playground.swift:592:20: error: cannot convert value of type 'ObjCBool' to expected argument type 'Bool'
            guard !isDirectory else { return }
                   ^~~~~~~~~~~
/root/toolbox/Sources/Cloud/Redis.swift:292:19: warning: 'catch' block is unreachable because no errors are thrown in 'do' block
                } catch {
                  ^
error: terminated(1): /usr/bin/swift-build-tool -f /root/toolbox/.build/release.yaml main output:


root@git:~/toolbox# git checkout 3.1.2
Previous HEAD position was d7b6a7b... Merge pull request #209 from vapor/fix-cloud-run
HEAD is now at 55229cc... Merge pull request #203 from vapor/feature/cleaning_up_db
root@git:~/toolbox# swift build -c release
Compile Swift Module 'Cloud' (68 sources)
/root/toolbox/Sources/Cloud/Commands/Configs/ConfigModify.swift:52:32: warning: 'characters' is deprecated: Please use String or Substring directly
            let parts = config.characters.split(separator: "=", maxSplits: 1)
                               ^
/root/toolbox/Sources/Cloud/Commands/Configs/ConfigModify.swift:60:22: warning: 'init' is deprecated: Please use String or Substring directly
                key: String(parts[0]),
                     ^
/root/toolbox/Sources/Cloud/Commands/Configs/ConfigModify.swift:61:24: warning: 'init' is deprecated: Please use String or Substring directly
                value: String(parts[1])
                       ^
/root/toolbox/Sources/Cloud/Commands/Configs/ConfigModify.swift:52:32: warning: 'characters' is deprecated: Please use String or Substring directly
            let parts = config.characters.split(separator: "=", maxSplits: 1)
                               ^
/root/toolbox/Sources/Cloud/Commands/Configs/ConfigModify.swift:60:22: warning: 'init' is deprecated: Please use String or Substring directly
                key: String(parts[0]),
                     ^
/root/toolbox/Sources/Cloud/Commands/Configs/ConfigModify.swift:61:24: warning: 'init' is deprecated: Please use String or Substring directly
                value: String(parts[1])
                       ^
/root/toolbox/Sources/Cloud/Commands/Database/DatabaseCreate.swift:45:9: warning: no calls to throwing functions occur within 'try' expression
        try environments.forEach { val in
        ^
/root/toolbox/Sources/Cloud/Commands/Database/DatabaseFeedbackFormat.swift:7:15: warning: 'catch' block is unreachable because no errors are thrown in 'do' block
            } catch {
              ^
/root/toolbox/Sources/Cloud/Commands/Database/DatabaseInspect.swift:46:9: warning: no calls to throwing functions occur within 'try' expression
        try environments.forEach { val in
        ^
/root/toolbox/Sources/Cloud/Commands/Playground.swift:592:20: error: cannot convert value of type 'ObjCBool' to expected argument type 'Bool'
            guard !isDirectory else { return }
                   ^~~~~~~~~~~
/root/toolbox/Sources/Cloud/Redis.swift:292:19: warning: 'catch' block is unreachable because no errors are thrown in 'do' block
                } catch {
                  ^
error: terminated(1): /usr/bin/swift-build-tool -f /root/toolbox/.build/release.yaml main output:
```

**Comment by bre7** [comment]
Fixed in #214 ?

**Comment by joscdk** [comment]
Yeah this should be solved.
